src/Security/Generator/JwtTokenGenerator.php

```diff
@@ -71,11 +72,11 @@ public function create(UserInterface $user, Request $request, string $subjectCla
             // jti claim
             ->identifiedBy($this->uuidFactory->uuid4()->toString())
             // iat claim
-            ->issuedAt($currentTime)
+            ->issuedAt((new DateTimeImmutable())->setTimestamp($currentTime))
             // nbf claim
-            ->canOnlyBeUsedAfter($currentTime)
+            ->canOnlyBeUsedAfter((new DateTimeImmutable())->setTimestamp($currentTime))
```

Comment:
Not that it reall matters but could you extract `(new DateTimeImmutable())->setTimestamp($currentTime)` from this line and line 75 into additional variable, please?